Makefile

```diff
@@ -137,14 +137,13 @@ docker-build:
 	docker build -t $(REPOSITORY):$(VERSION) .
 
 .PHONY: build
-build: manager manifests docker-build
```

Comment:
We intentionally no longer do a regular docker image build as a part of the `make build` procedure. This is to avoid building images multiple times unnecessarily in CI with the addition of the `docker-buildx` makefile target. 

Now, you must run `make build` and then `make docker-build` or `make docker-buildx` to build an image.